panel/views/pages/tree.php

```diff
@@ -53,6 +53,18 @@
                         <div class="page-route truncate mr-2" aria-hidden="true">
                             <span><?= $page->canonicalRoute() ?? $page->route() ?></span>
                         </div>
+                        <?php
+                            $imagePreviewFieldName = $page->scheme()->options()->get('imagePreviewField', '');
+                            $imagePreviewFieldValue = $imagePreviewFieldName == '' ? '' : $page->fields()->get($imagePreviewFieldName);
+                            $imagePreviewFile = $imagePreviewFieldValue == '' ? '' : $page->files()->get($imagePreviewFieldValue);
+                        ?>
+                        <?php if($imagePreviewFile != '') : ?>
+                            <div class="col-sm-1-2 col-xs-1-3 mt-3 pl-0">
```

Comment:
`.col-*` should always be wrapped in `.row` divs. This also takes care of the padding.

```suggestion
                            <div class="row mt-3">
                                <div class="col-sm-1-2 col-xs-1-3">
```